Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "parity-scale-codec"
 description = "SCALE - Simple Concatenating Aggregated Little Endians"
-version = "1.3.0"
+version = "1.3.1"
```

Comment:
We can not release this version. Master has breaking changes. If you want to release this. I propose that you backport this after merge to master. We have a tag for the last release.